src/createWidgetBase.ts

```diff
@@ -88,7 +98,7 @@ function dNodeToVNode(instance: Widget<WidgetState>, dNode: DNode): VNode | stri
 		return child.render();
 	}
 
-	dNode.children = dNode.children.map((child: DNode) => {
+	dNode.children = dNode.children.filter((child) => child).map((child: DNode) => {
```

Comment:
to match the `filter` elsewhere we should explicitly check for null, rather than falsy.